stablehlo/reference/Ops.cpp

```diff
@@ -42,25 +42,22 @@ std::vector<int64_t> permute(ArrayRef<int64_t> array, ArrayRef<int64_t> perm) {
 
 Tensor evalAddOp(const Tensor &lhs, const Tensor &rhs, Type resultType) {
   Tensor result(resultType);
-  for (auto it = result.index_begin(); it != result.index_end(); ++it) {
+  for (auto it = result.index_begin(); it != result.index_end(); ++it)
     result.set(*it, lhs.get(*it) + rhs.get(*it));
-  }
```

Comment:
We don't have a specific code style guide around whether or not to use braces for single-statement ifs and fors, but I think this change is fine. However, I would recommend to factor it out into a separate PR given that it ended up dominating the pull request.